assets/js/script.js

```diff
@@ -0,0 +1,337 @@
+
+  // create h1 title
+  const h1 = document.createElement("h1");
+  h1.setAttribute("class", "question-section-title");
+  h1.textContent = `Question ${questionIndex + 1}:`;
+
+  // creat h2 info
+  const h2 = document.createElement("h2");
+  h2.setAttribute("class", "question-section-info");
+  h2.textContent = question.question;
+
+  // creat ul info
+  const ul = document.createElement("ul");
+  ul.setAttribute("class", "answers");
+
+  // creat answers using forEach function
+  const createAppendAnswer = (answer) => {
+    const li = document.createElement("li");
+    const btn = document.createElement("button");
+    btn.setAttribute("class", "answer-button");
+    btn.setAttribute("data-value", answer);
+    btn.textContent = answer;
+    li.append(btn);
+    ul.append(li);
+  };
+
+  question.answers.forEach(createAppendAnswer);
+
+  // append section and main
+  questionSection.append(h1, h2, ul);
+  mainSection.append(questionSection);
+
+  // add click event listener on question-section to check answer
+  questionSection.addEventListener("click", validateAnswer);
+};
+
+const renderTimerSection = () => {
+  const timerSpanSection = document.createElement("section");
+  timerSpanSection.setAttribute("id", "timerSpanSection");
+  timerSpanSection.setAttribute("class", "timerSpanSection");
+  timerSpanSection.textContent = "Time Remaining: ";
+  const timerSpan = document.createElement("span");
+  timerSpan.setAttribute("id", "timerSpan");
+  timerSpan.textContent = timer;
+  timerSpanSection.append(timerSpan);
+  mainSection.append(timerSpanSection);
+
+  const startTimer = () => {
+    timer -= 1;
+    const timerSpan = document.getElementById("timerSpan");
+    timerSpan.textContent = timer;
+    if (timer === 0) {
+      //render gameover
+      clearInterval(timerId);
+    }
+    if (questionIndex === questions.length - 1) {
+      clearInterval(timerId);
+    }
+  };
+
+  //start the timer coountdown
+  const timerId = setInterval(startTimer, 1000);
+  // console.log(timerId);
+};
+
+const startQuiz = () => {
+  removeStartSection();
+
+  // render timerSection and starts the time counting down from 100 second
+  renderTimerSection();
+
+  // render question function
+  renderQuestionSection();
+};
+
+// add event listeners
+// add document on load event listener
+// add start button click event listener
+// window.addEventListener("load", onLoad);
+// add event listener to start button
+startBtn.addEventListener("click", startQuiz);
+
+// uncompleted functions??
+const onLoad = () => {
+  // initialise local storage
+  // check if highscores exists in LS
+  // if false then set highscores to empty array in LS
+};
+
+const handleFormSubmit = () => {
+  // get value from input
+  // check if empty then render error alert with message and status
+  // if not empty then create the score object
+  // {
+  //   fullName: "Bob Smith",
+  //   score: 25
+  // }
+  // push score object to LS
```

Comment:
If code isn't being used, please delete it rather than comment it out as it can make code look messy.